### ISSUE TITLE
[Feature Request] Read Go Book Format by smartgo

### ISSUE THREAD

**Issue by xiaodaigh** [Kashomon/glift#118]
http://www.smartgo.com/pdf/gobookformat.pdf

It would be great for Glift to support some book format. I think the current SGF format is hard to follow since to see all the branches we need to click each,back out and click the next. Go book formats allows more prose form of presenting a game so that branches can be presented linearly.

**Comment by Kashomon** [comment]
We have something we're working on for the publishing side of things that might solve this problem.
